arithmetization/src/main/java/net/consensys/linea/zktracer/module/add/Add.java

```diff
@@ -67,19 +67,19 @@ public List<ColumnHeader> columnsHeaders() {
   public void commit(List<MappedByteBuffer> buffers) {
     final Trace trace = new Trace(buffers);
     int stamp = 0;
-    for (AddOperation op : this.chunks) {
+    for (AddOperation op : chunks.getAll()) {
```

Comment:
we are with @OlivierBBB in war with `.this` and whenever we see it, delete it ;)

Reply:
True ☠️

Reply:
@ahamlat is there any benefit to keeping `this.bla` rather than writing `bla` directly ? When I look at the `.class` files it automatically inserts the omitted `this.`

Reply:
It is more for clarity especially when you have a parameter of a method with the same name as the class field. In this, this. is mandatory, but then you will need to add it everywhere to make it consistant. We can skip it if we are sure to not have to deal with this case.

Reply:
If there is a local variable with the same name of a class attribute, it is not only for clarity but that is how to refer to one or the other. I personally like using always `this` for class attributes since, if at some point, you declare a local variable with the same name then you'll change the behaviour in multiple places without having any feedback from Java.